### ISSUE TITLE
How to use Parallel node 

### ISSUE THREAD

**Issue by XingbangChan** [BehaviorTree/BehaviorTree.CPP#185]
hi, could you please provide an example of using parallel node ,all of your doc only provide the squence, fallback control node example. thx(^o^)／

**Comment by facontidavide** [comment]
I will... eventually

**Comment by gramss** [comment]
+1 from me. I would like to use this feature with ROS.

**Comment by Lecrapouille** [comment]
Yes, at least the doc should refer to its name, as well as ParallelAll, to indicate its existence. I guess this is 3thd most important control node in my opinion. @albuckley https://arxiv.org/pdf/1709.00084.pdf

**Comment by facontidavide** [comment]
Thanks for reporting, documentation fixed.
